client/src/components/clientlayout/forms/InputField.js

```diff
@@ -1,17 +1,19 @@
 import React from "react";
 export default class InputField extends React.Component {
-  OnEnter(event) {
+  OnEnter = event => {
     this.props.takeinput(event.target.value);
-  }
+  };
   render() {
     return (
       <div>
         <div>{this.props.name}</div>
         <div>
           <input
-            type="text"
+            type={this.props.type ? this.props.type : "text"}
```

Comment:
@priyanshkumar.i have already added type funtionality.check the code.

Reply:
You did not pass type as a prototype in FormInput.js so got confused.
Seems good 
great work.